python/tests/tests.yaml

```diff
@@ -27,6 +27,13 @@ perf-stock:
   volumes:
   - dst: /tests
     src: $PWD/python/tests
+xpu-base-layers-${PACKAGE_OPTION:-pip}:
```

Comment:
PACKAGE_OPTION is not a var, it should be IDP_VERSION

Reply:
made changes.